examples/stable-diffusion/text_to_image_inpainting.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env python
```

Comment:
This should be incorporated into `text_to_image_generation.py` sample as all SD samples are

Reply:
This patch has a dependency on [MC patch](https://github.com/huggingface/optimum-habana/pull/891) , once the mc patch is merged. I will update this patch